MASFoundation/Classes/models/MASSecurityConfiguration.h

```diff
@@ -21,6 +21,27 @@
 @interface MASSecurityConfiguration : MASObject
 
 
+/**
```

Comment:
What if the Customer wants to enable both intermediate and leaf cert pinning.
I think we need to ponder about clubbing them together with an 'OR' clause.

Reply:
then thats a subset if you want leaf you any way get the intermediate also today....so then only pinning on intermediate has no value

Reply:
What I meant it should be product management decision ?